backend/engine/plugins/trivy/main.py

```diff
@@ -1,114 +1,17 @@
-                    "component": component,
-                    "source": source,
-                    "id": vuln_id,
-                    "description": description_result.description,
-                    "severity": vuln.get("Severity", "").lower(),
-                    "remediation": description_result.remediation,
-                    "inventory": {
-                        "component": {
-                            "name": vuln.get("PkgName"),
-                            "version": vuln.get("InstalledVersion"),
-                            "type": component_type,
-                        },
-                        "advisory_ids": sorted(
-                            list(set(filter(None, [vuln_id, vuln.get("PrimaryURL")] + vuln.get("References", []))))
-                        ),
-                    },
-                }
-            )
-    return results
-
-
-def convert_type(component_type: str) -> str:
-    if component_type == "bundler":
-        return "gem"
-    return component_type.lower()
-
-
-def get_description_and_remediation(description, fixed_version) -> NamedTuple:
-    """
-    gets the description and remediation fields after pulling them from the vuln and appending/removing additional info
-    :param fixed_version:
-    :param description:
-    :return: NamedTuple containing the description and remediation
-    """
-    result = NamedTuple("DescriptionResult", [("description", str), ("remediation", str)])
-    if not description:
-        description = ""
-    remediation = ""
-    if DESC_REMEDIATION_SPLIT in description:
-        des_split = description.split(DESC_REMEDIATION_SPLIT)
-        remediation = des_split[1].strip()
-        description = des_split[0].strip()
-    if fixed_version:
-        remediation = f"Fixed Version: {fixed_version}. {remediation}".strip()
-    result.description = description
-    result.remediation = remediation
-    return result
-
-
-def convert_output(output_str: str):
-    if not output_str:
-        return None
-    try:
-        return json.loads(output_str)
-    except json.JSONDecodeError as e:
-        logger.error(e)
-        return None
-
-
-def execute_trivy_lock_scan(path: str):
-    proc = subprocess.run(["trivy", "-q", "fs", "-f", "json", path], capture_output=True, check=False)
-    if proc.returncode != 0:
-        logger.warning(proc.stderr.decode("utf-8"))
-        return None
-    if proc.stdout.decode("utf-8") == "null":
-        logger.warning("No response returned. No files to parse.")
-        return None
-    if proc.stdout and NO_RESULTS_TEXT not in proc.stdout.decode("utf-8"):
-        return proc.stdout.decode("utf-8")
-    logger.error(proc.stderr.decode("utf-8"))
-    return None
-
 
 def execute_trivy_image_scan(image: str):
-    proc = subprocess.run(["trivy", "-q", "image", "-f", "json", image], capture_output=True, check=False)
+    proc = subprocess.run(["trivy", "image", image, "--format", "json"], capture_output=True, check=False)
```

Comment:
Yea it does not omit anything on the actual results, it just gets rid of the log that trivy the tools itself returns when a scan is triggered which includes information like what package management they detected in the code base and how many language specific files they located. I thought it may be useful in debugging in the future to some how be able to extract those logs if ever needed to see more into what trivy is exactly finding/scanning, but I'm open to adding the quiet tag back if that is better practice.

Reply:
I think if we wanted to persist any of that output for debugging, we'd want to capture it in code and log it with `logger.debug()`. That said, I'm not sure it matters much either way if the command is run with the `quiet` flag or not as long as we're getting the output we need and there's no concern that the additional output would cause the container to hang or something. It just caught my eye since I wasn't sure the motivation.

Thanks for changing the args to use the non-abbreviated flags; I think this is good practice in code to make it more immediately understandable.